### DISCUSSION TITLE
PNP howto announcements and loose talks ;-)

### DISCUSSION THREAD

**Discussion by ernierasta** [ernierasta/pnp-howto#1] [General]
## 👋 Welcome!
Here we can discuss articles content, any comments, suggestions and questions are very welcomed!

**Comment by ernierasta** [comment]
Just to let you know. 
For now I am not promoting this article too much (it is mentioned on two threads on BGG forum). I am working on practical part how to Inkscape for creating component layouts now. When I will finish all my TODOs, I will start working on flooding BGG with links. :-D

My goal is to change PNP world, I want to have high-quality, easy to print PNP files available to all. And I know, that in many cases it lack of knowledge, not lack of will. Many PNP authors want to prepare their games better, but do not know how. Those articles are here to change that.

**Reply by jhauberg** [reply]
Very cool. I think this is already a good reference/guideline. Thanks for making it :)

**Comment by ernierasta** [comment]
Thank you for kind words! I saw you watching repo, great! It is very motivating, so ... Inkscape tutorial is there!
https://github.com/ernierasta/pnp-howto/blob/main/articles/inkscape/gutter-fold-without-template.md

**Comment by ernierasta** [comment]
Hi everyone,
I have just two things to say:

 * I have created **thread on BGG** to get more feedback, so if someone who doesn't have github account, but have one on BGG is reading this, you can go here: **https://boardgamegeek.com/thread/2557795/how-prepare-pnp-game-files-theory-and-practical-ho**
 * I am working on **automation tutorial**  - I am very excited about it as I realized - it is way to go generally. I believe that generating layouts and components is crucial to be able to provide good files for game, as it allows to do so many tricks with almost no additional work. I will push work in progress article soon, but I think it will go through few revisions and partial rewrites until it will be OK. My goal is to enable non-technical people to use automation without fear and show them, that every one can do that. And even for more technically skilled I hope to provide "real life" example how to. I was slightly delayed as I was working with Pele on few improvements to [countersheetsextension](https://github.com/lifelike/countersheetsextension) which I am using in tutorial. They allow me simplify it very considerably. I am done for now with my commits, I am just waiting for them to be merged.
 
So happy new year for everyone, I hope it will be less crazy then 2020!

Leszek

**Comment by ernierasta** [comment]
Hi everyone,
[automation guide](https://github.com/ernierasta/pnp-howto/blob/main/articles/inkscape/automation.md) starts to taking shape. I will probably put csv and special object names to separate articles.
Also, after finishing raw text I will section it better and add ton of illustrations to make it more understandable. 
Then I will link it from main article and put emphasis on it, as I am now using automation to many game creating/editing tasks and it is incredible help. Saves a lot of time and tedious work!

Stay safe and have great time!

Leszek
